src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/DetectorOptions.java

```diff
@@ -54,7 +54,8 @@
     private Pattern pattern;
 
     /** Default constructor.*/
-    private DetectorOptions() { }
+    private DetectorOptions() {
+    }
```

Comment:
please move this change to separate commit (first commit) of this PR.
Such change is ok even without fix in Check. It is ok to keep reference to the same issue as a reason of update.